ext/soap/php_sdl.c

```diff
@@ -2114,7 +2118,7 @@ static void add_sdl_to_cache(const char *fn, const char *uri, time_t t, sdlPtr s
 	WSDL_CACHE_PUT_N("wsdl", 4, out);
 	WSDL_CACHE_PUT_1(WSDL_CACHE_VERSION,out);
 	WSDL_CACHE_PUT_1(0,out);
-	WSDL_CACHE_PUT_N(&t, sizeof(t), out);
+	WSDL_CACHE_PUT_N((char*)&t, sizeof(t), out);
```

Comment:
was the cast needed ?

Reply:
I like this change: Casting in the `WSDL_CACHE_PUT_N` macro can hide errors. Casting here is more explicit.